src/translations/i18n.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+FICHIERS=$(ls -1 src/translations/*.json)
+OUTPUT_FILE=src/translations/index.js
+
+echo "/* eslint-disable */
+const TRANSLATIONS = {" > $OUTPUT_FILE
+
+for fichier in $FICHIERS; do
+    basename=${fichier##*/}
+    basename=${basename%.json}
```

Comment:
What kind of formatting is going on here? Maybe explain it in a comment?

Reply:
it's retrieving the basename without the file extension from the file path.
For example: `de` from `src/translations/de.json`.

Reply:
But, then you are adding `.json` again, so it's useless? Or don't I understand?

Reply:
starting from `src/translations/de.json`.
then basename = `de.json`
then basename = `de`